modules/overrides/user_configurable_overrides.go

```diff
@@ -24,6 +25,12 @@ import (
 	"github.com/grafana/tempo/tempodb/backend"
 )
 
+var metricUserConfigurableOverridesReloadFailed = promauto.NewCounter(prometheus.CounterOpts{
+	Namespace: "tempo",
```

Comment:
Yeah... I added metrics on the client that queries the backend, it must have been an oversight not to add one specifically for failed reloads.
We can already alert on the log lines but this will be cleaner.